Joshua branch start of portal set ups and hit effects added

JTAdams6501 opened the pull request:
ill add a fire hit effect soon once i finish the portals or if im needed i can also work on the trail again

JTAdams65 committed:
Smoke and trail

no grey bg anymore

JTAdams65 committed:
Merge branch 'main' into joshua_branch

JTAdams65 committed:
arrowtrail-prefab-fixes

did some fixes to help with quality of trails and game

JTAdams65 committed:
hit-effect

worked on some hit effects more

JTAdams65 committed:
organized some of the folders

JTAdams65 committed:
Particle hit

base hit effect let me know anything else to add

JTAdams65 committed:
Merge branch 'main' into joshua_branch

JTAdams65 committed:
edits on particles

JTAdams65 committed:
start of portal work

has started working with material and shader graph with the idea in mind

JTAdams65 committed:
Merge remote-tracking branch 'origin/main' into joshua_branch

JTAdams65 committed:
portal changes

start of adding the portal plan combine both brackeys tutorial and water shader

Colby626 reviewed (changes requested):
I like how the portal is coming along. For the hit effects. Hook it up in the Arrow script to spawn your particles whenever the arrow hits something (not the bow or the player but anything else). That way it will be easy to test and see how it looks in action.

JTAdams65 committed:
Portal effects implementation

JTAdams65 committed:
start of tele hit

if we added this version recommend playing with lifetimes/lifetimes of particles

JTAdams65 committed:
Merge remote-tracking branch 'origin/main' into joshua_branch

JTAdams65 committed:
tele hit effects

one of the two tele hit options

JTAdams65 committed:
Merge remote-tracking branch 'origin/main' into joshua_branch

JTAdams65 committed:
Arrow trail and minor portal changes

JTAdams65 committed:
Merge remote-tracking branch 'origin/main' into joshua_branch
